src/components/select/SingleSelectConnected.tsx

```diff
@@ -121,11 +121,17 @@ export class SingleSelectConnected extends React.Component<ISingleSelectProps &
 
     private getDeselectOptionButton(): React.ReactNode {
         return (
-            <Tooltip title={this.props.deselectTooltipText} placement='top' noSpanWrapper onClick={this.props.deselect}>
+            <Tooltip title={this.props.deselectTooltipText} placement='top' noSpanWrapper onClick={this.handleDeselect}>
                 <Svg svgName={VaporSVG.svg.clear.name} svgClass='icon mod-12' className='btn-append center-align' />
             </Tooltip>
         );
     }
+
+    private handleDeselect = () => {

```

Comment:
return your event to deselect if we do this

Reply:
The `deselect` prop isn't typed to accept an event as argument, and the event isn't used in the `mapDispatchToProps` anyway.